core/src/main/java/fr/sncf/osrd/conflicts/Conflicts.kt

```diff
@@ -4,8 +4,8 @@ import com.carrotsearch.hppc.IntArrayList
 import com.squareup.moshi.Json
 import fr.sncf.osrd.api.ConflictDetectionEndpoint.ConflictDetectionResult.Conflict
 import fr.sncf.osrd.api.ConflictDetectionEndpoint.ConflictDetectionResult.Conflict.ConflictType
-import fr.sncf.osrd.standalone_sim.result.ResultTrain.SpacingRequirement
```

Comment:
Ngl, i disagree. Otherwise we'll never clean up the imports. This should have been done in the PR modifying this file, absolutely, but it wasn't, and I'd rather correct it now than never.